frontend/src/components/GemHistory.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div id="timeline-wrapper">
+    <div id="timeline-svg-wrapper" ref="wrapper">
+      <inline-svg
```

Comment:
For those SVG files, we are loading them from the server so we are not loading the `.svg` filetype directly in the frontend. In the server, we are using the file system (`fs`) to read in these files and returning them.

If we want to load the `.svg.` filetype directly in the frontend, we need some type of file loader. Alternatively we could add this SVG file to the `api` instead and load it from the frontend. That would be one less library to include but I'm not sure if it would be better in terms of developer (more code needed) and user (potentially longer page load time) experiences.

Reply:
Thanks for the explanation @e0. I guess one option would be to serve this file from the API as well, but I'm not sure that would be an improvement in any way.